### PR TITLE
Bh feat 17

### PR DESCRIPTION
**Pull request by BethLambertMatthews**
Rspec passes - basics of Page 5 complete

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,15 +2,16 @@
...
   get "/game", to: "game#index"
-  get "/game/home", to: "game#home"
-  get "/game/right", to: "game#right"
-  get "/game/left", to: "game#left"
-  get "/game/page2", to: "game#page2"
+
   get "/game/page1", to: "game#page1"
   post "/game/page1", to: "game#page2"
+  # The post route above may not be needed if we change the button to a link?
+  get "/game/page2", to: "game#page2"
```
Comment (troysnowden) [inline config/routes.rb:9]: I think so Beth yes.

File: spec/features/player_can_use_page_5_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  #   visit ('game/page5')
+  #   click_link 'Wrong' 
+  #   expect(page).to have_current_path '/game/page6'
+  # end
+
+  # scenario "player moves to 'Win' when they click on the 'Correct' button" do
+  #   visit ('game/page5')
+  #   click_link 'Correct' 
+  #   expect(page).to have_current_path '/game/win'
+  # end
```
Comment (troysnowden) [inline spec/features/player_can_use_page_5_spec.rb:19]: Nice use of testing for different scenarios

**Review by troysnowden — APPROVED** [review]
Really good again :)

**Review by KirstyJHarris — APPROVED** [review]

**Comment by KirstyJHarris** [comment]
Great stuff :)

**Pull request merged**